src/Classes/ServiceAPI/MyRadio_List.php

```diff
@@ -345,6 +368,27 @@ public static function getAllLists()
         return $lists;
     }
 
+    /**
+     * Return mailing lists a user has access to.
+     *
+     * @return MyRadio_List[]
+     */
+    public static function getUserLists($memberid = -1)
```

Comment:
memberid undocumented

Reply:
Given my changes to getAllLists, I'm not actually going to use this. I'm wondering if we should just get rid of it.